cdtoolchainv2/cd_toolchain_v2.go

```diff
@@ -37,7 +36,7 @@ import (
 
 // CdToolchainV2 : This swagger document describes the options and endpoints of the Toolchain API.<br><br> All calls
 // require an <strong>Authorization</strong> HTTP header to be set with a bearer token, which can be generated using the
-// <a href="https://cloud.ibm.com/apidocs/iam-identity-token-api">Identity and Access Management (IAM)
+// <a href="https://cloud.ibm.com/toolchain/v2/apidocs/iam-identity-token-api">Identity and Access Management (IAM)
```

Comment:
This URL looks wrong, it should be
```suggestion
// <a href="https://cloud.ibm.com/apidocs/iam-identity-token-api">Identity and Access Management (IAM)
```

Reply:
@omar-albastami this needs to be fixed in the `otc-api` swagger

Reply:
Should be fixed @briangleeson